packages/w/wangeditor5.json

```
@@ -0,0 +1,35 @@
+{
+  "name": "wangEditor5",
+  "filename": "index.js",
+  "description": "wangEditor5, Open source web rich text editor, run right out of the box, config simply.",
+  "homepage": "https://www.wangeditor.com/",
+  "keywords": [
+    "wangEditor",
+    "editor",
+    "richi text editor"
+  ],
+  "repository": {
+    "type": "git",
+    "url": "https://github.com/wangeditor-team/wangEditor"
+  },
+  "license": "MIT",
+  "autoupdate": {
+    "source": "npm",
+    "target": "@wangeditor/editor",
+    "fileMap": [
+      {
+        "basePath": "dist",
+        "files": [
+          "**/*"
```

```suggestion
          "*.js?(.map)",
          "css/*.css"
```
We try to avoid using open globstars as they can result in matching files that we didn't intend to include.